### PR TITLE
ROGER-1000: Add act-as header on framework requests

### PR DESCRIPTION
**Pull request by 04msambit**
@amitbose327 Please review

### CODE REVIEW

**Comment by amitbose327** [comment]
LGTM

File: ansible/roles/nginx/templates/rogeros_auth_proxy.conf.j2 [hunk 1]
```diff
@@ -28,6 +28,7 @@ server {
 		auth_basic_user_file {{nginx_conf_dir}}/{{nginx_basic_auth_file_name}};
 		auth_request /auth-proxy;
                 proxy_pass http://marathon/;
```
Comment (ankanm) [inline ansible/roles/nginx/templates/rogeros_auth_proxy.conf.j2]: looks like a minor alignment issue in this block.. can you please fix it?

**Comment by ankanm** [comment]
LGTM apart from minor format issue.

File: ansible/roles/nginx/templates/rogeros_auth_proxy.conf.j2 (hunk 1 above)
Comment (04msambit) [inline ansible/roles/nginx/templates/rogeros_auth_proxy.conf.j2]: Done.

**Pull request merged**